packages/components/WalletProviderIcon.tsx

```diff
@@ -23,7 +24,14 @@ export const WalletProviderIcon: React.FC<{
     case WalletProvider.Adena:
       return <SVG width={size} height={size} source={adenaSVG} />;
     case WalletProvider.Gnotest:
-      return <SVG width={size} height={size} source={warningSVG} />;
+      return (
+        <SVG
+          width={size}
+          height={size}
+          source={warningSVG}
+          color={errorColor}
```

Comment:
should you use color & type: warning ? else do we use the warning type you add somewhere ?

Reply:
No.
I just set `color` here, because:
- warning.svg has now a dynamic `#color` instead of a static red
- So, when warning.svg is used, we need to set a `color` to `<SVG/>`.

I just needed to have a warning.svg with a dynamic `color`.

Now, I'll use this warning.svg with a `warningColor` in the NFT Launchpad PR.

Reply:
It's a common refacto: Instead of having many similar SVG with different colors, we can just have one SVG with a dynamic color prop.

We have a TON of SVGs in the assets folder

Reply:
Got it, LGTM :heart: